lib/api/wpc.h

```diff
@@ -39,49 +39,51 @@ typedef uint8_t net_channel_t;
  */
 typedef enum
 {
-    APP_ROLE_SINK = 1,    //!< Node is sink
-    APP_ROLE_HEADNODE = 2,//!< Node is headnode
-    APP_ROLE_SUBNODE = 3, //!< Node is subnode (no routing of other's traffic)
-    APP_ROLE_UNKNOWN      //!< Erroneous role
+    APP_ROLE_SINK = 1,        //!< Node is sink
+    APP_ROLE_HEADNODE = 2,    //!< Node is headnode
+    APP_ROLE_SUBNODE =
+        3,              //!< Node is subnode (no routing of other's traffic)
```

Comment:
Not a huge fan of this refactoring but I think only solution is to reduce a bit the comment length